hack/update_submodules.sh

```diff
@@ -39,7 +39,8 @@ function generate_version_commit() {
   if [ "$submodule" = "containerd" ]; then
     # Ref: Containerd Version https://github.com/containerd/containerd/blob/main/Makefile#L33
     version=$(git describe --match 'v[0-9]*' --dirty='.m' --always)
-  else
+  fi 
+  if [ "$submodule" = "kubelet" ] || [ "$submodule" = "kube-proxy" ]; then
```

Comment:
This is not the correct fix. `generate_version_commit` should not be called for `csi-proxy` in the first place. Please see line 96.